### PR TITLE
Updated the ReadMe

### PR DESCRIPTION
**Pull request by amura11**

### CODE REVIEW

**Comment by arifogel** [comment]
After I finally merged my pull request to fix issues with XText later than 2.7, you're proposed README is out of date. Can you update this? I am currently running with latest version of XText. Note that this will still be an issue until the marketplace release is updated, so I will wait to merge this until then.

**Comment by arifogel** [comment]
The marketplace release has been updated. There is no longer any reason to separately install an old version of xtext. Please update your pull request, or I will close and reject in a couple weeks or so.

File: README.md [hunk 1]
```diff
@@ -34,76 +34,143 @@ The main reason of complexity of the old IDE was in Dynamic Language ToolKit (DL
...
-* Go to: ```Help > Eclipse Marketplace...```
-* Search for ```antlr```
-* Choose ```ANTLR 4 IDE``` and click ```Install```
+Prerequisites
+---------
 
+* Eclipse 4.4 Luna Xtext Complete SDK(Needs to be version 2.7.3)
+* Eclipse Faceted Project Framework (Tested with 3.4.0) Eclipse Faceted
+* Project Framework JDT Enablement(Tested with 3.4.0) ANTLR 4 SDK A
+* A copy of the antlr-4.x-complete.jar (4.5 at the time of writing)
```
Comment (HSorensen) [inline README.md]: pls add the antlr.org download link

**Comment by arifogel** [comment]
That makes sense. I think @jknack will have to start it, if it's not
already enabled.

**Comment by timeraider4u** [comment]
Maybe it would make sense to create a community-project for Antlr4IDE so that multiple users/developers can take over responsibilities and/or manage Github features.

**Comment by HSorensen** [comment]
I will accept this PR and start the Wiki Pages based on some of this content.

**Pull request merged**